workbench-core/accounts/src/models/costCenters/costCenter.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { z } from 'zod';
+
+export interface CostCenter {
```

Comment:
I'd recommend exporting CostCenter using the `type CostCenter = z.infer<typeof CostCenterParser>` so we don't need to do double declaration of the cost structure properties in an interface and in the parser

Reply:
Good call, updating